python/image-content-search/stack/cdk.py

```diff
@@ -336,7 +346,7 @@ def __init__(self, scope: Construct, id: str, **kwargs) -> None:
 
         ### outputs
         CfnOutput(self, 'CognitoHostedUILogin',
-            value='https://{}.auth.{}.amazoncognito.com/login?client_id={}&response_type=token&scope={}&redirect_uri={}'.format(user_pool_domain.domain_name, Aws.REGION, user_pool_app_client.ref, '+'.join(user_pool_app_client.allowed_o_auth_scopes), api_gateway.url_for_path('/web')),
+            value='https://{}.auth.{}.amazoncognito.com/login?client_id={}&response_type=token&scope={}&redirect_uri={}'.format(user_pool_domain.domain_name or "", Aws.REGION, user_pool_app_client.ref, '+'.join(user_pool_app_client.allowed_o_auth_scopes or []), api_gateway.url_for_path(api_gateway_landing_page_resource.path)),
```

Comment:
Rewrite for clarity.

Reply:
Is the following an acceptable rewrite, or did you have something else in mind?

```suggestion
            value='https://{domain}.auth.{region}.amazoncognito.com/login?client_id={client_id}&response_type=token&scope={scope}&redirect_uri={redirect_uri}'.format(
                domain=user_pool_domain.domain_name or "",
                region=Aws.REGION,
                client_id=user_pool_app_client.ref,
                scope='+'.join(user_pool_app_client.allowed_o_auth_scopes or []),
                redirect_uri=api_gateway.url_for_path(api_gateway_landing_page_resource.path)
            ),
```

Reply:
Lines split up for readability and compared to original cdk synth output for correctness